pipeline_steps/aio_prepare.groovy

```diff
@@ -2,7 +2,7 @@ def prepare(){
   common.conditionalStage(
     stage_name: "Prepare Deployment",
     stage: {
-      if (env.STAGES.contains("Upgrade")) {
+      if ((env.STAGES.contains("Major Upgrade")) || (env.STAGES.contains("Leap Upgrade"))) {
```

Comment:
I'd prefer to see `leapfrog` used throughout these changes instead of `leap`, there's no need the scrimp on characters and I'd rather the name was consistent.